### PR TITLE
Masters

### PR DESCRIPTION
**Pull request by fredrikved**
This implements series-masters and other-masters.

It adds the file, defines the filename and sets meta based on the properties in the order form.

Yet to be extensively tested, and the audio analyze should be finalized.

### CODE REVIEW

File: workflows/ingest/masters.go [hunk 1]
```diff
@@ -96,14 +106,127 @@ func VBMaster(ctx workflow.Context, params VBMasterParams) (*VBMasterResult, err
...
+	}
+	for _, key := range keys {
+		err = wfutils.SetVidispineMeta(ctx, assetID, key, strconv.FormatFloat(values[key], 'f', 2, 64))
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	return &result, nil
+}
```
Comment (fredrikved) [inline workflows/ingest/masters.go:189]: @KillerX Can you take a look at this?

Comment (KillerX) [reply workflows/ingest/masters.go:189]: Put -24 as the target. The analyze part in general does not really care about the target, it's only used to calc suggested adjustment. The Values in 171+ seem correct.

**Review by KillerX — APPROVED** [review]
LGTM.

Do we already have a way of doing notifications?

**Comment by fredrikved** [comment]
> LGTM.
> 
> Do we already have a way of doing notifications?

No.

Haven't implemented that anywhere.

**Pull request merged**